src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -933,6 +939,10 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
     }
   }, [handleSubmit, isBelowMinimumWithdrawAmount])
 
+  const handleAsymSideChange = useCallback((asymSide: string | null) => {
+    setOpportunityType(asymSide as AsymSide | 'sym')
+  }, [])
```

Comment:
And update types in `<AddLiquidityInput />` and `<LpType />` while at it? 🙏🏽 

```suggestion
  const handleAsymSideChange = useCallback((asymSide: OpportunityType | null) => {
    if (!asymSide) return
    setOpportunityType(asymSide)
  }, [])
```

Reply:
Changed it to 

```
  const handleAsymSideChange = useCallback((asymSide: string | null) => {
    setWithdrawType(asymSide as OpportunityType)
  }, [])
  ```
  
Because this handler is used inside a `useRadioGroup` which require callback taking a string as a parameter, adding a safeguard feel a bit useless as these radios are constructed with `OpportunityType` but as you feel!

Inside [3f46c39](https://github.com/shapeshift/web/pull/7351/commits/3f46c39c62ae8b335139b436a73c7c22405becd5)